tests/core/channels/test_twilio_voice.py

```diff
@@ -0,0 +1,206 @@
+
+    twiml = tv.build_twilio_voice_response(output_channel.messages)
+    assert (
+        str(twiml)
+        == '<?xml version="1.0" encoding="UTF-8"?><Response><Say voice="woman">Buttons:</Say><Pause length="1" /><Say voice="woman">Yes</Say><Pause length="1" /><Gather action="/webhooks/twilio_voice/webhook" actionOnEmptyResult="true" enhanced="false" input="speech" speechModel="default" speechTimeout="5"><Say voice="woman">No</Say></Gather></Response>'
+    )
+
+
+@pytest.mark.parametrize(
+    "configs, expected",
+    [
+        (
+            {
+                "initial_prompt": "hello",
+                "reprompt_fallback_phrase": "i didn't get that",
+                "speech_model": "default",
+                "speech_timeout": "5",
+                "assistant_voice": "alien",
+                "enhanced": "false"
+            }, InvalidConfigException
+        ),
+        (
+            {
+                "initial_prompt": "hello",
+                "reprompt_fallback_phrase": "i didn't get that",
+                "speech_model": "default",
+                "speech_timeout": "not a number",
+                "assistant_voice": "woman",
+                "enhanced": "false"
+            }, InvalidConfigException
+        ),
+        (
+            {
+                "initial_prompt": "hello",
+                "reprompt_fallback_phrase": "i didn't get that",
+                "speech_model": "default",
+                "speech_timeout": "auto",
+                "assistant_voice": "woman",
+                "enhanced": "wrong"
+            }, InvalidConfigException
+        ),
+        (
+            {
+                "initial_prompt": "hello",
+                "reprompt_fallback_phrase": "i didn't get that",
+                "speech_model": "default",
+                "speech_timeout": "5",
+                "assistant_voice": "woman",
+                "enhanced": "true"
+            }, InvalidConfigException
+        )
+    ]
+)
+def test_invalid_configs(configs, expected):
+    with pytest.raises(expected):
+        TwilioVoiceInput(**configs)
+
+
+async def test_twilio_voice_remove_image():
+
+    with pytest.warns(UserWarning):
+        output_channel = TwilioVoiceCollectingOutputChannel()
+        await output_channel.send_response(
+            recipient_id="Chuck Norris",
+            message={
+                "image": "https://i.imgur.com/nGF1K8f.jpg",
+                "text": "Some text."
+            }
+        )
+        assert len(output_channel.messages) == 1
+        assert output_channel.messages[0]["text"] == "Some text."
+
+
+async def test_twilio_emoji_warning():
+
+    with pytest.warns(UserWarning):
+        output_channel = TwilioVoiceCollectingOutputChannel()
+        await output_channel.send_response(
+            recipient_id="User",
+            message={
+                "text": "Howdy 😀"
+            }
+        )
+
+
+def test_twilio_incompatible_speech_timeout_and_model():
```

Comment:
can we please add these two to `test_invalid_configs`? Or use `pytest.mark.parametrize` here?